### PR TITLE
- disable mdadm auto assemble during DASD dialog

### PR DESCRIPTION
**Pull request by aschnell**
Cherry-picked from SLE-15-GA branch.

For https://trello.com/c/sBGUAwc8/374-sles15-p2-1089645-build5671s390x-unable-to-format-dasd-which-contains-a-linux-raid-partition.

### CODE REVIEW

**Comment by jreidinger** [comment]
code-wise it looks good. I am just a bit scaried by mentioned cherry-picking. Why not simple merge of SLE-15-GA branch as documented at http://yastgithubio.readthedocs.io/en/latest/maintenance-branches/ ?

**Comment by aschnell** [comment]
I do not know the difference between the two methods. I just picked one.

**Comment by jreidinger** [comment]
@aschnell difference is quite big. Cherry-pick basically took commit and commit it as different commit. It is good when commit is backporting and not whole branch should be branched. So e.g. when fix is done in master and it needs to be backported to SLE-15-GA branch. But here it is second case. What we want is to ensure that all fixes in SLE-15-GA branch is in master. So for this purpose we require to use git merge on whole branch. This way it is easy to verify with git tools, that all fixes in maintenance branch is included in master.

So in short:

1. if fix is done in maintenance branch, use git merge for merge to newer branch.
2. if fix is done in newer branch and need to be backported, then cherry-pick should be used.

**Comment by aschnell** [comment]
I followed the workflow now and get changes in Dockerfile and Rakefile. I do not know how to not include those in the PR.

**Comment by jreidinger** [comment]
@aschnell mark them as not wanted with `git revert <commit number>` This is needed to be done only once, next git merge knows that this commit is already reverted as not something for master.
It is mentioned in given doc as `# then use git revert <commit number> and next time it will not appear`. If it is not clear enough, I welcome any proposal how to improve it. Thanks

**Comment by aschnell** [comment]
I had to read that several times to understand it. Maybe just mention Dockerfile and Rakefile as examples for specific commits.
